### PR TITLE
Add Gitlab Syntax For RSS Feed

### PR DESCRIPTION
**Pull request by matchboxbananasynergy**
This PR adds the necessary syntax to track app updates on Gitlab for android apps. It also removes the screenshots of gfeeds showing the RSS feed as I believe we should eventually replace those with screenshots of an Android RSS reader such as Feeder or Read You, as that makes a lot more sense (you should be tracking app updates with an android RSS feeder so that you can click and download the update on that same device).

- [x] Add light and dark mode screenshot of an Android RSS feeder to demonstrate what such a feed would look like in practice.

### CODE REVIEW

File: docs/android.en.md [hunk 1]
```diff
@@ -320,12 +320,19 @@ To mitigate these problems, we recommend [Neo Store](https://github.com/NeoAppli
 
 ### Manually with RSS Notifications
 
-If an app is released on a platform like GitHub, you may be able to add an RSS feed to your [news aggregator](/news-aggregators) that will help you be aware of new releases. Using [Secure Camera](#secure-camera) as an example, you would navigate to its [releases page](https://github.com/GrapheneOS/Camera/releases) on GitHub and append `.atom` to the URL:
+For apps that are released on platforms like Github and Gitlab, you may be able to add an RSS feed to your [news aggregator](/news-aggregators) that will help you keep track of new releases.
+
+#### Github
```
Comment (mfwmyfacewhen) [inline docs/android.en.md:325]:
```suggestion
#### GitHub
```

File: docs/android.en.md [hunk 2]
```diff
@@ -320,12 +320,19 @@ To mitigate these problems, we recommend [Neo Store](https://github.com/NeoAppli
 
 ### Manually with RSS Notifications
 
-If an app is released on a platform like GitHub, you may be able to add an RSS feed to your [news aggregator](/news-aggregators) that will help you be aware of new releases. Using [Secure Camera](#secure-camera) as an example, you would navigate to its [releases page](https://github.com/GrapheneOS/Camera/releases) on GitHub and append `.atom` to the URL:
+For apps that are released on platforms like Github and Gitlab, you may be able to add an RSS feed to your [news aggregator](/news-aggregators) that will help you keep track of new releases.
```
Comment (mfwmyfacewhen) [inline docs/android.en.md:323]:
```suggestion
For apps that are released on platforms like GitHub and GitLab, you may be able to add an RSS feed to your [news aggregator](/news-aggregators) that will help you keep track of new releases.
```

File: docs/android.en.md [hunk 3]
```diff
@@ -320,12 +320,19 @@ To mitigate these problems, we recommend [Neo Store](https://github.com/NeoAppli
...
+
+#### Github
+
+On Github, using [Secure Camera](#secure-camera) as an example, you would navigate to its [releases page](https://github.com/GrapheneOS/Camera/releases) and append `.atom` to the URL:
```
Comment (mfwmyfacewhen) [inline docs/android.en.md:327]:
```suggestion
On GitHub, using [Secure Camera](#secure-camera) as an example, you would navigate to its [releases page](https://github.com/GrapheneOS/Camera/releases) and append `.atom` to the URL:
```

File: docs/android.en.md [hunk 4]
```diff
@@ -320,12 +320,19 @@ To mitigate these problems, we recommend [Neo Store](https://github.com/NeoAppli
...
 
 `https://github.com/GrapheneOS/Camera/releases.atom`
 
-![RSS Feed](./assets/img/android/gfeeds-light.png#only-light)
-![RSS Feed](./assets/img/android/gfeeds-dark.png#only-dark)
+#### Gitlab
```
Comment (mfwmyfacewhen) [inline docs/android.en.md:331]:
```suggestion
#### GitLab
```

File: docs/android.en.md [hunk 5]
```diff
@@ -320,12 +320,19 @@ To mitigate these problems, we recommend [Neo Store](https://github.com/NeoAppli
...
+
+On Gitlab, using [Aurora Store](#aurora-store) as an example, you would navigate to its [project repository](https://gitlab.com/AuroraOSS/AuroraStore) and append `/-/tags?format=atom` to the URL:
```
Comment (mfwmyfacewhen) [inline docs/android.en.md:333]:
```suggestion
On GitLab, using [Aurora Store](#aurora-store) as an example, you would navigate to its [project repository](https://gitlab.com/AuroraOSS/AuroraStore) and append `/-/tags?format=atom` to the URL:
```

**Review by mfwmyfacewhen — CHANGES_REQUESTED** [review]

**Review by mfwmyfacewhen — APPROVED** [review]

**Review by jonaharagon — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Comment by dngray** [comment]
@matchboxbananasynergy if you think that looks okay we can merge.

**Comment by matchboxbananasynergy** [comment]
> @matchboxbananasynergy if you think that looks okay we can merge.

LGTM, we can merge

**Pull request merged**